deploy/crds/aws.managed.openshift.io_accountclaims_crd.yaml

```diff
@@ -22,11 +22,12 @@ spec:
     listKind: AccountClaimList
     plural: accountclaims
     singular: accountclaim
-  scope: Namespaced
+  scope: Cluster
```

Comment:
Hm, this doesn't seem like a valid change. Aren't account claims scoped to the namespace of the clusterdeployment?